### ISSUE TITLE
Issue while installing the package MSI of the release 1.6.3 (Windows)

### ISSUE THREAD

**Issue by cstisa** [crowdsecurity/crowdsec#3223]
### What happened?

Dear all,

When installing the package crowdsec_1.6.3.msi using msiexec.exe (via Ansible win_package module) or simply manually via the command line prompt using `msiexec.exe /a "crowdsec_1.6.3.msi" /qn /L*V "crowdsecinstall.txt"` 

It does not complete as it say that there is no sufficient priviliges while stopping or starting the Crowdsec windows service.

`MSI (s) (DC:34) [09:32:16:825]: Product: CrowdSec -- Error 1920. Service 'Crowdsec' (Crowdsec) failed to start.  Verify that you have sufficient privileges to start system services.`

### What did you expect to happen?

That the MSI installation complete successfuly.

### How can we reproduce it (as minimally and precisely as possible)?

- Install version of Crowdsec 1.6.2 and upgrade by installing Crowdsec 1.6.3
- Install Crowdsec 1.6.3 using GUI
- Install Crowdsec 1.6.3 using msiexec 

### Crowdsec version

<details>
</details>

### OS version

<details>

```console
# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
BuildNumber  Caption                                 OSArchitecture  Version
17763        Microsoft Windows Server 2019 Standard  64-bit          10.0.17763
```

</details>

### Enabled collections and parsers

<details>

```console
$ cscli hub list -o raw
# paste output here
```

</details>

### Acquisition config

<details>
```console
# On Linux:
$ cat /etc/crowdsec/acquis.yaml /etc/crowdsec/acquis.d/*
# paste output here

# On Windows:
C:\> Get-Content C:\ProgramData\CrowdSec\config\acquis.yaml
# paste output here
</details>


### Config show

<details>

```console
$ cscli config show
# paste output here
```

</details>

### Prometheus metrics

<details>

```console
$ cscli metrics
# paste output here
```

</details>

### Related custom configs versions (if applicable) : notification plugins, custom scenarios, parsers etc.

<details>

</details>

Thanks a lot for the support !

Best Regards, Edouard Fazenda for CSTI SA.

**Comment by blotus** [comment]
Hello,

Do you have any errors in `%PROGRAMDATA%\CrowdSec\logs\crowdsec.log` ?

Was it a clean install or an upgrade from 1.6.2 ?

I'm going to try to reproduce the issue on my side.

**Comment by cstisa** [comment]
Dear @blotus ,

No errors in the `%PROGRAMDATA%\CrowdSec\logs\crowdsec.log` log file.

It was from a upgrade from 1.6.2, but i also tried a clean install of crowdsec 1.6.3 and the issue is also present.

Thanks for the help.

Best Regards.

**Comment by lhari84** [comment]
Hi,

Same here on all 3 different Windows Servers. I've tried everything, even uninstalling both bouncer and main application and reinstalling. I've even deleted all CrowdSec config files and folders left in file system after uninstall - no success. 

Windows Event Viewer Logs just say: 
The Crowdsec service failed to start due to the following error: 
The service did not respond to the start or control request in a timely fashion.

CrowdSec itself will not even write logs to %PROGRAMDATA%\CrowdSec\logs\, after fresh installation and failing to start the service during setup process. 

During fresh installation after uninstall, it just stucks at the same point as when failing to update: 

![image](https://github.com/user-attachments/assets/41e6e850-3de9-40f0-932a-8bf843ddd3e6)

**Comment by blotus** [comment]
Hello,

We have found the cause of this issue, it's related to one of our dependency (https://github.com/wasilibs/go-re2/issues/144). 
We have submitted a patch upstream to resolve this, but in the meantime we'll try to find a workaround for the issue.

We have also removed the 1.6.3 MSI from the release, we will republish it once we have a fix.

**Comment by blotus** [comment]
Hello,

I have re-released the 1.6.3 version for windows, and the crash is resolved.
Please try it, and let us know if some find any other issues.

**Comment by cstisa** [comment]
@blotus All ok on our side , thanks a lot !
